### PR TITLE
Revised Two Topics to Account for the Access Log Download Report

### PR DESCRIPTION
**Pull request by marcfishelquantil**
The addition of this report required edits to two topics.

### CODE REVIEW

File: docs/portal/reports.md [hunk 1]
```diff
@@ -40,6 +41,7 @@ The following report types are supported:
 | --------------- | ----------------------------------------------------- |
 | Report Type     | Select the type of report you want to generate.       | 
 | Date Range      | Select the start and end dates and the time for the report. |
+| Hostname | For Access Log Download reports, select a property hostname. Logs related to visitors associated with that hostname will be generated for you to view and download.
```
Comment (burtonquantil) [inline docs/portal/reports.md:44]: I suggest rephrasing "Logs related to visitors associated with that hostname"   to "Logs of visitors to the hostname"
would be more succinct.

File: docs/portal/reports.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ The following report types are supported:
 | Traffic Volume Summary| Summarizes the edge traffic volume.                               |
 | CPU Time| Reports the CPU time consumed in each time interval to serve your properties. You can toggle the unit between **seconds** and **number of cores**. For example, if 3000 seconds of CPU time are reported in a 5-minute interval, it means your properties are maxing out an average of 10 (=3000s/5m) physical CPU cores during that period.                               |
 | Edge Hostname Request| Shows the number of requests made to your [edge hostnames](</docs/portal/traffic-management/creating-edge-hostname.md>) over a period of time.                               |
+| Access Log Download| Shows what content has been requested and when it was requested, the IP address from which requests were made, the value of the User-Agent HTTP request header, and the size of the content requested.                    |
```
Comment (burtonquantil) [inline docs/portal/reports.md:31]: Once we get precise details of the log file format, we'll elaborate on it further.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I made a small suggestion.

File: docs/portal/reports.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/reports.md:44]: Sounds good. I will make the fix and re-upload.

File: docs/portal/reports.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/portal/reports.md:31]: Confirmed.

**Review by burtonquantil — APPROVED** [review]
OK to merge.

**Pull request merged**